spec/orm/activerecord_spec.rb

```diff
@@ -185,29 +185,57 @@ class Event < ActiveRecord::Base; end # setup a basic AR class for testing
     end
 
     describe '#image=' do
-
-      it "should cache a file" do
+      it "caches a file" do
         @event.image = stub_file('test.jpeg')
         expect(@event.image).to be_an_instance_of(@uploader)
       end
 
-      it "should write nothing to the database, to prevent overriden filenames to fail because of unassigned attributes" do
+      it "writes nothing to the database, to prevent overriden filenames to fail because of unassigned attributes" do
         expect(@event[:image]).to be_nil
       end
 
-      it "should copy a file into into the cache directory" do
+      it "copies a file into into the cache directory" do
         @event.image = stub_file('test.jpeg')
         expect(@event.image.current_path).to match(%r(^#{public_path('uploads/tmp')}))
       end
 
-      it "should do nothing when nil is assigned" do
-        @event.image = nil
-        expect(@event.image).to be_blank
+      context "when empty string is assigned" do
+        it "does nothing when" do
+          @event.image = ''
+          expect(@event.image).to be_blank
+        end
+
+        context "and the previous value was an empty string" do
+          before do
+            @event.image = ""
```

Comment:
Perhaps you can this guy above to a before block inside the context `when empty string is assigned` and get a little dry. What do you think?